plugins/development/plugin-git/src/client/styles/changelog.scss

```diff
@@ -0,0 +1,153 @@
+
+    font-weight: bold;
+
+    cursor: pointer;
+
+    @media (max-width: 419px) {
+      flex-wrap: wrap;
+    }
+
+    .vpi-changelog,
+    .vpi-changelog-menu {
+      display: inline-block;
+      vertical-align: middle;
+
+      width: 1.2em;
+      height: 1.2em;
+      margin-right: 4px;
+
+      background-color: currentcolor;
+      color: var(--vp-c-text-mute);
+
+      mask: var(--icon) no-repeat;
+      mask-size: 100% 100%;
+
+      transition: color var(--vp-t-color);
+
+      transform: translateY(-1px);
+    }
+
+    .vpi-changelog {
+      --icon: url("data:image/svg+xml,%3Csvg xmlns='http://www.w3.org/2000/svg' width='1em' height='1em' viewBox='0 0 16 16'%3E%3Cpath fill='none' stroke='%23000' stroke-linecap='round' stroke-linejoin='round' d='M2.71 10.96a6.5 6.5 0 1 0-.69-3.53M2 8l1.5-1.5M2 8L.5 6.5m8 2v-4m0 4h3'/%3E%3C/svg%3E");
+    }
+
+    .vpi-changelog-menu {
+      --icon: url("data:image/svg+xml,%3Csvg xmlns='http://www.w3.org/2000/svg' width='1em' height='1em' viewBox='0 0 24 24'%3E%3Cg fill='none' stroke='%23000' stroke-linecap='round' stroke-linejoin='round' stroke-width='2'%3E%3Cpath d='M4 5h0.01'/%3E%3Cpath d='M8 5h12'/%3E%3Cpath d='M4 10h0.01'/%3E%3Cpath d='M8 10h12'/%3E%3Cpath d='M4 15h0.01'/%3E%3Cpath d='M8 15h12'/%3E%3Cpath d='M4 20h0.01'/%3E%3Cpath d='M8 20h12'/%3E%3C/g%3E%3C/svg%3E");
+    }
+  }
+
+  .changelog-list {
+    display: none;
+    margin: 0.5rem 0;
+    padding-left: 0;
+    list-style: none;
+  }
+
+  &.active {
+    padding: 0 1rem 0.5rem;
+
+    .changelog-header {
+      padding-bottom: 0.5rem;
+    }
+
+    .changelog-list {
+      display: block;
+    }
+  }
+
+  .changelog {
+    position: relative;
+    margin-top: 8px;
+    padding-left: 20px;
+
+    &:first-child {
+      margin-top: 0;
+    }
+
+    &::before {
+      content: '';
+
+      position: absolute;
+      top: 3px;
+      left: 0;
+
+      display: inline-block;
+
+      width: 1.25em;
+      height: 1.25em;
+
+      background-color: currentcolor;
+      color: var(--vp-c-text-subtle);
+
+      mask: var(--icon) no-repeat;
+      mask-size: 100% 100%;
+
+      transition: color var(--vp-t-color);
+    }
+
+    &.commit::before {
+      --icon: url("data:image/svg+xml,%3Csvg xmlns='http://www.w3.org/2000/svg' width='1em' height='1em' viewBox='0 0 24 24'%3E%3Cpath fill='none' stroke='%23000' stroke-linecap='round' stroke-linejoin='round' stroke-width='2' d='M9 12a3 3 0 1 0 6 0a3 3 0 1 0-6 0m3-9v6m0 6v6'/%3E%3C/svg%3E");
+    }
+
+    &.release-tag::before {
+      --icon: url("data:image/svg+xml,%3Csvg xmlns='http://www.w3.org/2000/svg' width='1em' height='1em' viewBox='0 0 24 24'%3E%3Cg fill='none' stroke='%23000' stroke-linecap='round' stroke-linejoin='round' stroke-width='2'%3E%3Cpath d='M6.5 7.5a1 1 0 1 0 2 0a1 1 0 1 0-2 0'/%3E%3Cpath d='M3 6v5.172a2 2 0 0 0 .586 1.414l7.71 7.71a2.41 2.41 0 0 0 3.408 0l5.592-5.592a2.41 2.41 0 0 0 0-3.408l-7.71-7.71A2 2 0 0 0 11.172 3H6a3 3 0 0 0-3 3'/%3E%3C/g%3E%3C/svg%3E");
+    }
```

Comment:
I would suggest removing the width and height attribute in svg. The `--icon` shall be designed to work with any size of svg, user are not expected to set their own svg in some "correct size"

 I see `mask-size: 100% 100%;` is already added, then `width='1em' height='1em'` should be removed here.